p2p/stream/protocols/sync/protocol.go

```diff
@@ -180,7 +180,8 @@ func (p *Protocol) HandleStream(raw libp2p_network.Stream) {
 			Msg("failed to add new stream")
 		return
 	}
-	fmt.Println("Connected to", raw.Conn().RemotePeer().String(), "(", st.ProtoID(), ")", "my ID: ", raw.Conn().LocalPeer().String())
+	//to get my ID use raw.Conn().LocalPeer().String()
+	fmt.Println("Connected to", raw.Conn().RemotePeer().String(), "(", st.ProtoID(), ")")
```

Comment:
change it to utils.logging() ?

Reply:
better remove it

Reply:
why is that ? and different it is between the p2p connected/disconnected message ? not saying we should add it, but we should at least be consistent.

Reply:
Actually this line was one of the most useful lines to debug and trace the connectivity issues. That's how I could check the node connections. The idea of logging connection is common on many existed blockchains. The reason why we haven't have this connection lines is because we were using the DNS nodes which are trusted nodes. I can move this line to log file if preferred.

Reply:
yeah agree i'd move this to the log